content/smart.md

```diff
@@ -154,7 +156,17 @@ SMART apps can be embedded and launched from the *MPages* Workflow or Summary. I
 
 #### iframe Cookies ####
 
-If your SMART app uses cookies, the [P3P](https://www.w3.org/P3P/) header or meta tag must be set in order for cookies to be saved correctly while running in an *MPages* iframe.  Please see [this page](https://blogs.msdn.microsoft.com/ieinternals/2013/09/17/a-quick-look-at-p3p/) for more information on P3P.
+For IE if your SMART app uses cookies, the [P3P](https://www.w3.org/P3P/) header or meta tag must be set in order for cookies to be saved correctly while running in an *MPages* iframe.  Please see [this page](https://blogs.msdn.microsoft.com/ieinternals/2013/09/17/a-quick-look-at-p3p/) for more information on P3P.
+
+For Edge if your SMART app uses cookies, the `Set-Cookie` response header has to contain the attributes: `SameSite=None; Secure` for your cookies to be saved correctly while running in an iframe.
+
+### Microsoft Edge Browser (Chromium based) Support ###
+We are excited to share that there is full support for embedded Microsoft Edge browser (Chromium based) inside *Cerner Millennium PowerChart*!
+
+With Microsoft Edge as an additional option, clients can configure any SMART app to load using Edge (where available). Based on our test results, SMART apps render faster, use less CPU, and less memory to run in comparison to IE. If your app uses cookies while being loaded inside an iframe, or is embedded in *MPages* (which loads the app in an iframe) you would need to set the `Set-Cookie` response header with additional attributes: `SameSite=None; Secure` for your cookies to be persisted.
```

Comment:
`and less memory to run in comparison to IE.` => `and less memory to run, in comparison to IE.`